### PR TITLE
fix(rns): adds post find pagination hook

### LINKED ISSUE #528: Re-enable pagination for RNS offers
https://github.com/rsksmart/rif-marketplace-cache/pull/516 removes pagination using feathers due to an issue this had caused.
Pagination should be rebuild either by sql queries or by finding a solution to using feathers pagination without causing the mentioned error again.

### PR DESCRIPTION
**Pull request by jurajpiar**
Part resolves https://github.com/rsksmart/rif-marketplace-cache/issues/528
with https://github.com/rsksmart/rif-marketplace-ui/pull/697

### CODE REVIEW

**Review by nduchak — APPROVED** [review]
Nice job. 👍

**Comment by jurajpiar** [comment]
Need to add correct size of the filtered results. Currently the total is set to all results.

**Comment by jurajpiar** [comment]
> Need to add correct size of the filtered results. Currently the total is set to all results.

done by using user-defined page limit in the after hook

**Review by julianmrodri — APPROVED** [review]
LGTM, UI also Ok we can proceed to merge both.

**Review by nduchak — APPROVED** [review]

File: src/services/rns/hooks/domain-offer.hooks.ts [hunk 1]
```diff
@@ -151,7 +162,27 @@ export default {
 
   after: {
     all: [],
-    find: [],
+    find: [
+      (context: HookContext): HookContext => {
```
[the remark this answers is not in the document]
Comment (julianmrodri) [reply src/services/rns/hooks/domain-offer.hooks.ts]: @jurajpiar any comments on this? I would like to get this functionality in if possible..

[the remark this answers is not in the document]
Comment (jurajpiar) [reply src/services/rns/hooks/domain-offer.hooks.ts]: @julianmrodri this has been done. Just forgot to "Resolve conversation"

**Comment by julianmrodri** [comment]
@nduchak  can we merge? @jurajpiar once its merged lets do UI as well.

**Pull request merged**